GitCommands/RevisionGraph.cs

```diff
@@ -98,12 +98,21 @@ public RevisionGraph(GitModule module)
 
         ~RevisionGraph()
         {
-            Dispose();
+            Dispose(false);
         }
 
         public void Dispose()
         {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        private void Dispose(bool disposing)
+        {
             _backgroundLoader.Cancel();
```

Comment:
`_backgroundLoader` is a managed resource, and can only be used if `disposing` is true.